concent_api/api-integration-force-accept-or-reject-results-test.py

```diff
@@ -262,38 +255,35 @@ def main():
         PROVIDER_PRIVATE_KEY,
         CONCENT_PUBLIC_KEY,
         headers = {
-            'Content-Type': 'application/octet-stream',
+            'Content-Type'             : 'application/octet-stream',
             'concent-client-public-key': b64encode(PROVIDER_PUBLIC_KEY).decode('ascii'),
-        }
+        },
+        expected_status = 200
     )
 
     # Test CASE 2D + 3 + 4B + 5. Requestor sends ForceSubtaskResultsResponse with SubtaskResultsRejected
+    task_2 = create_task(task_to_compute_timestamp, task_id + '2D', compute_task_def_deadline)
```

Comment:
Each parameter should be in separate line.

Reply:
@rwrzesien , there is no such recommendation in our coding style guide. Why would you liketo have it like that?

Reply:
@dybi Once upon a time we had a discussion about it and @cameel told me he will add it there :)

Reply:
I was objecting to something like this:
``` python
task_2 = create_task(
    task_to_compute_timestamp, task_id + '2D', compute_task_def_deadline
)
```
because it suggests that it's a single parameter. But this:

``` python
task_2 = create_task(task_to_compute_timestamp, task_id + '2D', compute_task_def_deadline)
```
is typical and completely fine. It's still hard to distinguish the parameters but it does not mislead you into thinking it's a single one when you're just skimming through the code.

About the style guide: I'm sorry, I have a list of minor stuff that came up in reviews to add there. I was a bit too overloaded recently to update it but I'm going to get to if after the talk about style we have planned.